index.css

```diff
@@ -0,0 +1,186 @@
+body {
+    background-color: #f2f2f2;
+    color: #333;
+    font-weight: 400;
+    line-height: 1.3;
+    margin: 32px;
+    font-family: Arial, Arial, Helvetica, sans-serif;
+}
+
+img {
+    max-width: 100%;
+}
+
+.nav {
+    box-sizing: border;
+    padding: 15px;
+}
+
+.topnav {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+
+.topnav a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+.u-list-reset {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+}
+
+.title {
+    box-sizing: border;
+    padding: 10px;
+    margin-top: -30px;
+    margin-bottom: -10px;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+
+.big1 {
+    font-size: 50px;
+    background-color: #333;
+    width: 18%;
+    color: white;
+}
+
+.big2 {
+    font-size: 50px;
+    background-color: #333;
+    width: 15%;
+    color: white;
+}
+
+.h1title {
+    padding-bottom: 20px;
+    border-bottom: 3px solid #22333B;
+}
+
+.strong a {
+    font-family: Arial, Helvetica, sans-serif;
+    color: #333;
+    text-decoration: underline overline;
+}
+
+.author-img {
+    border-radius: 50%;
+    height: auto;
+    margin-right: auto;
+    margin-bottom: 12px;
+    margin-left: auto;
+    width: 120px;
+}
+
+.author-company {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 15px;
+    font-weight: 300;
+    text-transform: uppercase;
+}
+
+.speakers,
+.speakers fotter {
+    border-radius: 0 0 -6px 6px;
+}
+
+.speakers footer {
+    background-color: #22333B;
+    color: white;
+    margin: 20px -12px 10px;
+    padding: 12px;
+    text-align: left;
+}
+
+.speaker-title {
+    box-sizing: border;
+    margin-top: -140px;
+    margin-left: 140px;
+    text-align: left;
+    color: white;
+}
+
+.speakers {
+    background-image: url("https://cdn.glitch.com/46be344e-bd86-488e-bf75-ab4ed5228fee%2Fbackground.png?v=1608233769495");
+    background-position: center;
+    margin-bottom: 24px;
+    padding: 12px;
+    transition: all 300ms ease-in;
+}
+
+.speakers {
+    background-size: 30%;
+}
+
+.speakers a {
+    color: white;
+    text-decoration: none;
+}
+
+.speakers a:hover {
+    text-decoration: wavy underline;
+}
+
+.register {
+    text-align: center;
+    padding: 20px;
+    color: white;
+    background-color: #333;
+    margin-left: 5px;
+    margin-right: 5px;
+    font-size: 16px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+@media (max-width: 575px) {
+    .author-img {
+        display: inline-block;
+        vertical-align: middle;
+        margin-bottom: auto;
+    }
+
+    .speakers footer {
+        font-size: 12px;
```

Comment:
This text size is too small for information you actually want conference attendees to read and benefit from. In general, body text on the web should be 16-18px.